src/App/Fossa/Analyze.hs

```diff
@@ -123,6 +141,53 @@ analyze basedir destination override unpackArchives = runFinally $ do
             ]
           traverse_ (\err -> logError $ "FOSSA error: " <> viaShow err) (uploadError resp)
 
+          contribResult <- Diag.runDiagnostics $ runExecIO $ tryUploadContributors basedir baseurl apiKey $ uploadLocator resp
+          case contribResult of
+            Left failure -> logDebug (Diag.renderFailureBundle failure)
+            Right _ -> pure ()
+
+tryUploadContributors ::
+  ( Has Diag.Diagnostics sig m
+  , Has Exec sig m
+  , MonadIO m
+  ) => Path x Dir
+  -> URI
+  -> ApiKey
+  -> Text -- ^ Locator
+  -> m ()
+tryUploadContributors baseDir baseUrl apiKey locator = do
+  contributors <- fetchGitContributors baseDir
+  uploadContributors baseUrl apiKey locator contributors
+
+
+fetchGitContributors :: 
```

Comment:
[nit] can we move this to another module? (along with `gitLogCmd`, of course)

Reply:
Have a name/existing module in mind?

Reply:
I'd be okay with a new module named `VCS.Git` or similar